app/index.js

```diff
@@ -120,6 +120,20 @@ render(
       </Provider>
     </ErrorBoundary>
     {DevTools && <DevTools position={{ bottom: 0, right: 0 }} />}
-  </div>,
+  </React.Fragment>,
   document.getElementById('root')
 );
+
+window.addEventListener('load', async () => {
+  // installation does not use Google Analytics, or tracking is blocked on client
+  // no point loading the rest of the analytics bundles
+  if (!process.env.GOOGLE_ANALYTICS_ID || !window.ga) return;
+
+  await import('autotrack/lib/plugins/outbound-link-tracker');
```

Comment:
Additional analytics code is async loaded, only if the browser hasn't blocked GA. So that's pretty nifty.